### PR TITLE
unpin nbqa pre-commit hook's additional dependencies

### PR DESCRIPTION
**Pull request by andersy005**
- Addresses https://github.com/ProjectPythia/pythia-foundations/pull/257#issuecomment-1095429683

### CODE REVIEW

File: foundations/quickstart.ipynb [hunk 1]
```diff
@@ -219,7 +219,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "type(0.1)"
```
Comment (andersy005) [inline foundations/quickstart.ipynb:222]: it appears that `pyupgrade` has broken this line

File: foundations/quickstart.ipynb [hunk 2]
```diff
@@ -265,7 +265,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "type(\"orange\")"
```
Comment (andersy005) [inline foundations/quickstart.ipynb:268]: it appears that `pyupgrade` has broken this line

File: foundations/quickstart.ipynb [hunk 3]
```diff
@@ -275,7 +275,7 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "type('orange')"
```
Comment (andersy005) [inline foundations/quickstart.ipynb:278]: it appears that `pyupgrade` has broken this line

**Comment by andersy005** [comment]
the link checker failures are unrelated: 

- #260

**Comment by kmpaul** [comment]
These link-checker errors are all for valid external links.  I don't have time to look deeply into this at the moment.  I'm wondering if anyone has any idea why this is failing?

**Pull request merged**